keyboards/anavi/knobs3/config.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 1
+#define MATRIX_COLS 3
+
```

Comment:
```suggestion
```

Reply:
Hi @fauxpark,

If we remove these constants from `config.h`, we will need to use the real values to the keymaps in:

```
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    [_BASE] = LAYOUT_k3(
        KC_MUTE, RGB_TOG, RGB_MOD
    )
};
```
I don't insist but I prefer to keep the constants. However, it is your call.

Thanks,
Leon

Reply:
The CLI can derive these from the structure of `DIRECT_PINS` (as you're using here), or the size of `MATRIX_ROW_PINS`/`MATRIX_COL_PINS`, and generates them automatically during compilation.

Reply:
This is really nice feature. I didn't know it. Thank you for letting me know. I will make the change and force push the updated commit.